data/gegno/gegno intro missions.txt

```diff
@@ -439,6 +439,168 @@ mission "Giaru Gegno: Revisit"
+			choice
+				`	(Leave the planet.)`
+					goto "leave"
+				`	(Prepare for combat.)`
+			`	Engaging your weapon systems, you brace yourself for a hostile encounter as they approach. The Scin ship advances towards yours then slowly comes to a hault in front of you. They haven't fired any weapons, and they aren't displaying any aggressive behavior, so they might not be looking for conflict. Ever so slightly, the warship reverses backwards, then comes to a stop. You decide to stop your ship as well, wondering what this is about. Once again, the Scin ship reverses again, slowly drifting away from you. It appears they want you to follow them.`
+			choice
+				`	(Follow.)`
+					goto "follow"
+				`	(Leave.)`
+					goto "leave"
+			label "leave"
+			`	Wary of the Scin warship, you turn around and begin to ascend back into space. The Scin ship advances towards yours, but as it comes closer, it begins to take on an escort formation with you, following you as you leave the planet's atmosphere. Once you finally make it back into space, the Scin ship breaks from formation and flies away. You're unsure of what they're doing on the planet, but it's probably better you are careful not to get involved.`
+				decline
+
+			label "follow"
+			`	You inch your ship forward in acknowledgment. All of a sudden, the second warship on the ground lifts off and forms up behind you, while one in front of you turns around then begins to fly off into the distance. You follow them, curious of where they might be leading you. To your rear, the remaining shuttle takes off, but instead of following you, it heads off into the atmosphere above.`
+			`	Although this planet initially appeared dull due to the several layers of thick clouds, the surface shows occasional signs of life such as plains of grass or occasional dried-up riverbeds, while patches of low fog cover various other locations. As the flight continues, the two Scin ships slowly start to move around so that one is on either side of you. Up ahead in the distance, a large structure comes into view from behind another wall of fog. After getting a bit closer to it, you make out the familiar green colors of the ships flying with you now. The two warships begin to descend towards what you assume to be a sizeable Scin freighter sitting in a large open area of sand.`
+			`	Your previous encounter with a Vi Warror was chaotic and unpredictable, so it's best to be extra cautious in this situation. As you set your ship down near the vessel, the two warships that guided you here also land nearby and power down. Other than the Gegno ships in front of you, there doesn't seem to be any strange activity on your radar at the moment. Environmental readings of the outside surprisingly indicate conditions safe for a human, but it's better to be safe than sorry. You equip your environment suit and your laser pistol just in case.`
```

Comment:
```suggestion
			`	Your previous encounter with a Vi Warror was chaotic and unpredictable, so it would be best to be extra cautious in this situation. As you set your ship down near the vessel, the two warships that guided you here also land nearby and power down. Other than the Gegno ships in front of you, there doesn't seem to be any strange activity on your radar at the moment. Environmental readings of the outside surprisingly indicate conditions safe for a human, but it's better to be safe than sorry. You equip your environment suit and your laser pistol just in case.`
```
"it might be" would also work here.

Reply:
"It might be" would work well. "It would be" sounds like you're telling the player what they're thinking.